src/scripts/load.py

```diff
@@ -0,0 +1,75 @@
+"""
+Script to be run to insert existing data from a spreadsheet to the database.
+"""
+
+import csv
+from affiliations.models import Affiliation, Submitter, Coordinator
+
+
+def run():  # pylint: disable-msg=too-many-locals
+    """Iterate through CSV and create Affiliation, Submitter ID, and Coordinator
+    objects in the DB."""
+
+    with open(
+        "/Users/gabriellasanchez/Desktop/repos/stanford-affils/src/scripts/test_affils.csv",
+        encoding="utf-8",
+    ) as file:
+        csv_reader = csv.DictReader(file)
+
+        Affiliation.objects.all().delete()
```

Comment:
Actually, this feels like a side effect. I'm not sure this should go in the `run` function. Maybe define a separate function `purge_all_affils` or something along those lines, and call that before calling this `run` function. That way it's super obvious to the reader what's happening when you run the script.

Reply:
Removed this line per the convo above.